diff --git a/general.ent b/general.ent
index d417de0..89a2e32 100644
--- a/general.ent
+++ b/general.ent
@@ -1,16 +1,16 @@
-<!ENTITY version         "8.4">
-<!ENTITY short-version   "8.4">  <!-- Used below in &blfs-book; 
+<!ENTITY version         "9.0">
+<!ENTITY short-version   "9.0">  <!-- Used below in &blfs-book; 
                                       Change to x.y for release but not -rc releases -->
-<!ENTITY generic-version "8.4"> <!-- Use "development"  or "x.y[-pre{x}]" -->
+<!ENTITY generic-version "9.0"> <!-- Use "development"  or "x.y[-pre{x}]" -->
 
-<!ENTITY versiond        "8.4-systemd">
+<!ENTITY versiond        "9.0-systemd">
 <!ENTITY short-versiond  "systemd">
 <!ENTITY generic-versiond "systemd"> 
 
-<!ENTITY releasedate     "Maarch 1, 2019">
+<!ENTITY releasedate     "September 1st, 2019">
 
 <!ENTITY copyrightdate   "1999-2019"><!-- jhalfs needs a literal dash, not &ndash; -->
-<!ENTITY milestone       "8.4">
+<!ENTITY milestone       "9.0">
 
 <!ENTITY lfs-root        "http://www.linuxfromscratch.org/">
 <!ENTITY blfs-root       "&lfs-root;blfs/">
