diff --git a/INSTALL b/INSTALL
index a67f534..182598e 100644
--- a/INSTALL
+++ b/INSTALL
@@ -25,7 +25,7 @@ If you want to convert the XML to HTML, install the following:
   - http://www.linuxfromscratch.org/blfs/view/svn/pst/docbook-xsl.html
 
 * HTMLTidy
-  - http://www.linuxfromscratch.org/blfs/view/svn/general/tidy.html
+  - http://www.linuxfromscratch.org/blfs/view/svn/general/tidy-html5.html
 
 ------------------------------------------------------------------------------- 
 
