diff --git a/make-aux-files.sh b/make-aux-files.sh
index fbec733..ad8bcea 100644
--- a/make-aux-files.sh
+++ b/make-aux-files.sh
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-rm -f lfs-bootscripts*.tar.bz2
+rm -f lfs-bootscripts*.tar.?z*
 
 # Get base file name and move bootscripts directory to that name
 version=`grep "ENTITY lfs-bootscripts-version" packages.ent |cut -d'"' -f2`
 mv bootscripts lfs-bootscripts-$version
 
 # Create the tarball and clean up
-tar -cjf lfs-bootscripts-$version.tar.bz2 --exclude .svn lfs-bootscripts-$version
+tar -cJf lfs-bootscripts-$version.tar.xz --exclude .svn lfs-bootscripts-$version
 mv lfs-bootscripts-$version bootscripts 
 
 #rm -f udev-config*.bz2
