diff --git a/patches.ent b/patches.ent
index 461646a..480d0d2 100644
--- a/patches.ent
+++ b/patches.ent
@@ -48,21 +48,15 @@
 <!ENTITY kbd-backspace-patch "kbd-&kbd-version;-backspace-1.patch">
 <!ENTITY kbd-backspace-patch-md5 "f75cca16a38da6caa7d52151f7136895">
 <!ENTITY kbd-backspace-patch-size "12 KB">
-
+<!--
 <!ENTITY ninja-limit-jobs-patch "ninja-&ninja-version;-add_NINJAJOBS_var-1.patch">
 <!ENTITY ninja-limit-jobs-patch-md5 "f537a633532492e805aa342fa869ca45">
 <!ENTITY ninja-limit-jobs-patch-size "2.5 KB">
-
-<!ENTITY sysvinit-consolidated-patch "sysvinit-&sysvinit-version;-consolidated-1.patch">
-<!ENTITY sysvinit-consolidated-patch-md5 "b7872b43a97a81bac4f9fc8ffaef4364">
-<!ENTITY sysvinit-consolidated-patch-size "2.5 KB">
-
-<!-- 
-<!ENTITY systemd-security-patch "systemd-&systemd-version;-security_fix-1.patch">
-<!ENTITY systemd-security-patch-md5 "aa9d81fec9a3875a54e63af86904513a">
-<!ENTITY systemd-security-patch-size "4 KB">
 -->
+<!ENTITY sysvinit-consolidated-patch "sysvinit-&sysvinit-version;-consolidated-1.patch">
+<!ENTITY sysvinit-consolidated-patch-md5 "aaa84675e717504d7d3da452c8c2eaf1">
+<!ENTITY sysvinit-consolidated-patch-size "2.6 KB">
 
-<!ENTITY systemd-glibc-patch "systemd-&systemd-version;-glibc_statx_fix-1.patch">
-<!ENTITY systemd-glibc-patch-md5 "a44057d6a6ca69760bafc126458d89e5">
-<!ENTITY systemd-glibc-patch-size "4 KB">
+<!ENTITY systemd-security-patch "systemd-&systemd-version;-security_fixes-2.patch">
+<!ENTITY systemd-security-patch-md5 "10abebce8ff5d9fd402623ace39b5ab8">
+<!ENTITY systemd-security-patch-size "14 KB">
