diff --git a/stylesheets/lfs-xsl/chunkfast.xsl b/stylesheets/lfs-xsl/chunkfast.xsl
index 5b94668..1582c8e 100644
--- a/stylesheets/lfs-xsl/chunkfast.xsl
+++ b/stylesheets/lfs-xsl/chunkfast.xsl
@@ -2,7 +2,7 @@
 
 <!--
 $LastChangedBy: manuel $
-$Date: 2007-07-05 18:18:33 -0300 (Thu, 05 Jul 2007) $
+$Date: 2007-07-05 14:18:33 -0700 (Thu, 05 Jul 2007) $
 -->
 
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
