diff --git a/stylesheets/lfs-xsl/chunk-slave.xsl b/stylesheets/lfs-xsl/chunk-slave.xsl
index 00c4fe9..e69012a 100644
--- a/stylesheets/lfs-xsl/chunk-slave.xsl
+++ b/stylesheets/lfs-xsl/chunk-slave.xsl
@@ -2,7 +2,7 @@
 
 <!--
 $LastChangedBy: bdubbs $
-$Date: 2015-08-10 14:53:57 -0300 (Mon, 10 Aug 2015) $
+$Date: 2015-08-10 10:53:57 -0700 (Mon, 10 Aug 2015) $
 -->
 
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
