diff --git a/stylesheets/lfs-xsl/nochunks.xsl b/stylesheets/lfs-xsl/nochunks.xsl
index 479ef86..7d2f6af 100644
--- a/stylesheets/lfs-xsl/nochunks.xsl
+++ b/stylesheets/lfs-xsl/nochunks.xsl
@@ -2,7 +2,7 @@
 
 <!--
 $LastChangedBy: matthew $
-$Date: 2013-10-08 13:03:29 -0700 (Tue, 08 Oct 2013) $
+$Date: 2013-10-09 04:03:29 +0800 (Wed, 09 Oct 2013) $
 -->
 
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
