diff --git a/stylesheets/lfs-xsl/profile.xsl b/stylesheets/lfs-xsl/profile.xsl
index 4e5d6dd..ebe7d7c 100644
--- a/stylesheets/lfs-xsl/profile.xsl
+++ b/stylesheets/lfs-xsl/profile.xsl
@@ -2,7 +2,7 @@
 
 <!--
 $LastChangedBy: matthew $
-$Date: 2013-10-08 17:03:29 -0300 (Tue, 08 Oct 2013) $
+$Date: 2013-10-08 13:03:29 -0700 (Tue, 08 Oct 2013) $
 -->
 
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
