diff --git a/stylesheets/lfs-xsl/xhtml/lfs-legalnotice.xsl b/stylesheets/lfs-xsl/xhtml/lfs-legalnotice.xsl
index c4eaeef..d9f3ed4 100644
--- a/stylesheets/lfs-xsl/xhtml/lfs-legalnotice.xsl
+++ b/stylesheets/lfs-xsl/xhtml/lfs-legalnotice.xsl
@@ -2,7 +2,7 @@
 
 <!--
 $LastChangedBy: manuel $
-$Date: 2007-07-05 14:18:33 -0700 (Thu, 05 Jul 2007) $
+$Date: 2007-07-06 05:18:33 +0800 (Fri, 06 Jul 2007) $
 -->
 
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
