diff --git a/stylesheets/lfs-xsl/xhtml/lfs-index.xsl b/stylesheets/lfs-xsl/xhtml/lfs-index.xsl
index d2b5954..f421850 100644
--- a/stylesheets/lfs-xsl/xhtml/lfs-index.xsl
+++ b/stylesheets/lfs-xsl/xhtml/lfs-index.xsl
@@ -2,7 +2,7 @@
 
 <!--
 $LastChangedBy: matthew $
-$Date: 2013-10-10 07:46:50 +0800 (Thu, 10 Oct 2013) $
+$Date: 2013-10-10 07:46:50 +0800 (周四, 10 10月 2013) $
 -->
 
 <!DOCTYPE xsl:stylesheet [
