diff --git a/bootscripts/ChangeLog b/bootscripts/ChangeLog
index 9600652..5a809d1 100644
--- a/bootscripts/ChangeLog
+++ b/bootscripts/ChangeLog
@@ -1,3 +1,11 @@
+2015-06-24 Bruce Dubbs <bdubbs@linuxfromscratch.org>
+   * Unmount network file systems before bringing down the network
+   * Spell out some mount/umount options
+
+2017-03-25 DJ Lucas <dj@linuxfromscratch.org>
+   * Bash-4.4 changes no longer allow continue in function to pass to calling
+     loop.
+
 2015-02-22 Bruce Dubbs <bdubbs@linuxfromscratch.org>
    * Cosmetic changes to network scripts' output
 
