diff --git a/stylesheets/lfs-xsl/pdf/lfs-index.xsl b/stylesheets/lfs-xsl/pdf/lfs-index.xsl
index ab54554..9038a3c 100644
--- a/stylesheets/lfs-xsl/pdf/lfs-index.xsl
+++ b/stylesheets/lfs-xsl/pdf/lfs-index.xsl
@@ -2,7 +2,7 @@
 
 <!--
 $LastChangedBy: bdubbs $
-$Date: 2012-08-29 17:45:23 -0300 (Wed, 29 Aug 2012) $
+$Date: 2012-08-29 13:45:23 -0700 (Wed, 29 Aug 2012) $
 -->
 
 <!DOCTYPE xsl:stylesheet [
