diff --git a/stylesheets/lfs-xsl/pdf/lfs-mixed.xsl b/stylesheets/lfs-xsl/pdf/lfs-mixed.xsl
index 9685250..abcccc7 100644
--- a/stylesheets/lfs-xsl/pdf/lfs-mixed.xsl
+++ b/stylesheets/lfs-xsl/pdf/lfs-mixed.xsl
@@ -2,7 +2,7 @@
 
 <!--
 $LastChangedBy: bdubbs $
-$Date: 2011-10-09 23:59:11 -0300 (Sun, 09 Oct 2011) $
+$Date: 2011-10-09 19:59:11 -0700 (Sun, 09 Oct 2011) $
 -->
 
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
